### PR TITLE
Update React newsletters section

### PR DESCRIPTION
**Pull request by slorber**
The React newsletters section is not very up-to-date

### CODE REVIEW

**Review by slorber — COMMENTED** [review]
Added some comments to help review

File: README.md [hunk 1]
```diff
@@ -110,10 +110,11 @@ Thanks to all [contributors](https://github.com/zudochkin/awesome-newsletters/gr
 
 #### React
 
-- [ReactJS Newsletter](http://reactjsnewsletter.com/). The free, weekly newsletter of the best React.js news and articles.
-- [React Native Newsletter](http://reactnative.cc/). Free occasional summary of React Native news, articles, issues & pull requests, libraries and apps.
+- [This Week In React](https://thisweekinreact.com). Weekly newsletter for experienced React and React-Native developers.
```
[the remark this answers is not in the document]
Comment (ferhatelmas) [reply README.md:113]: I would say why not since you have improved it a bit, this could be our appreciation for your help 😄

**Review by ferhatelmas — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (slorber) [reply README.md:113]: thanks :D

**Pull request merged**